lib/resources.js

```diff
@@ -65,7 +65,7 @@ const SEARCH_SCOPES = {
     fields: ['seriesStatement.folded']
   },
   callnumber: {
-    fields: ['shelfMark']
+    fields: ['shelfMark', 'items.shelfMark']
```

Comment:
This is not strictly related to this fix. It just adds `items.shelfMark` as one of the fields to match on when doing a search with search_scope=callnumber, which no one uses right now. (But may soon.)